src/v/config/configuration.cc

```diff
@@ -1376,6 +1376,21 @@ configuration::configuration()
       "performance",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , cloud_storage_start_upload_loop(
+      *this,
+      "cloud_storage_start_upload_loop",
```

Comment:
Maybe these new configs should be called differently to make it more obvious that the user is not expected to change them.

Reply:
Maybe also invert them so both properties would be 'disable_xxx' and will be false by default. Maybe this way it will be a bit more clear.

Reply:
Done and done